src/ui/app/redux/slices/sui-objects/index.ts

```diff
@@ -266,6 +259,10 @@ const initialState = objectsAdapter.getInitialState<SuiObjectsManualState>({
     objectResponses: [],
 });
 
+// This is probably dangerous and should be removed as soon a possible
```

Comment:
This is a potential blocker. Whatever is going on in here is complicated but the error is vague and the manual testing does not blow up. I haven't tried to figure this out yet, and don't know how long/hard it would be to figure out.